README.md

```diff
@@ -1,10 +1,9 @@
 # curriculum-databases-projects-template
+.
+## ScreenShot of queries made
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
-## Getting Started
+![screenshot](./images/screenshot1.png)
+![screenshot](./images/screenshot2.png)
```

Comment:
- [ ] As the previous reviewer has requested, kindly remove these command line screenshots from your ReadMe as they're causing cluttering and it does not look very professional